doc/classes/Dictionary.xml

```diff
@@ -4,16 +4,63 @@
 		Dictionary type.
 	</brief_description>
 	<description>
-		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Dictionary type. Associative container which contains values referenced by unique keys. Just like in real life each dictionary composes multiple pairs of unique keys and values. You can define a dictionary by placing a comma separated list of key:value pairs in curly braces [code]{}[/code].
```

Comment:
I'd drop "Just like in real life", real life dictionaries are "word: multiple descriptions", which are similar but not just like "arbitrary key: arbitrary value" pairs used in GDScript.
Also "is composed of".
```suggestion
		Dictionary type. Associative container which contains values referenced by unique keys. Dictionary are composed of pairs of keys (which must be unique) and values. You can define a dictionary by placing a comma separated list of [code]key: value[/code] pairs in curly braces [code]{}[/code].
```